### DISCUSSION TITLE
Parse examples in CI

### DISCUSSION THREAD

**Discussion by ObserverOfTime** [tree-sitter-grammars/.github#4] [Ideas]
Add a step to the CI workflow that parses files in an `examples` directory, if it exists.

**Comment by clason** [comment]
(Use submodules if we want to parse an actual corpus, like luv for Lua or nvim-treesitter for query?)

**Reply by ObserverOfTime** [reply]
This could be done by developing a JavaScript action so we can also have globs on Windows
and using `actions/checkout` to pull examples since submodules don't support sparse checkout.

**Reply by clason** [reply]
See https://github.com/sogaiu/ts-questions

**Comment by clason** [comment]
The goal is to have consistent workflows for _all_ parsers under the org.

**Comment by ObserverOfTime** [comment]
https://github.com/tree-sitter-grammars/tree-sitter-examples-action
